docs/index.md

```diff
@@ -106,7 +106,7 @@ Versions in Linux distributions:
 * [1.4.3](Release-Notes-1.4.3) (2017-08-7)
 * [1.4.2](Release-Notes-1.4.2) (2017-06-20)
 * [1.4.1](Release-Notes-1.4.1) (2017-04-26)
-* [1.3.5](Release-Notes-1.3.5) - only for EL6
+* [1.3.5](Release-Notes-1.3.5) - only for EL6; change path to “df” in hw-info plugin
```

Comment:
Can you please specify the date in brackets: AuthorDate: Thu Mar 2 11:15:54 2017 +0100

Reply:
nitpick: change the path to the "df" binary